ciw/import_params.py

```diff
@@ -72,6 +76,10 @@ def create_network(arrival_distributions=None,
         params['ps_thresholds'] = ps_thresholds
     if server_priority_functions != None:
         params['server_priority_functions'] = server_priority_functions
+    if reneging_time_distributions != None:
+        params['reneging_time_distributions'] = reneging_time_distributions
+    if reneging_destinations != None:
+        params['reneging_destinations'] = reneging_destinations
```

Comment:
```suggestion
    if reneging_time_distributions is not None:
        params['reneging_time_distributions'] = reneging_time_distributions
    if reneging_destinations is not None:
        params['reneging_destinations'] = reneging_destinations
```

https://appdividend.com/2022/01/10/python-check-none/